string name                          # object's name
float32 velocity                     # new velocity
--- 
